### DISCUSSION TITLE
Deta login don't work in brave

### DISCUSSION THREAD

**Discussion by Fenriuz** [deta/cloud-docs#315] [Help]
![image](https://user-images.githubusercontent.com/63080658/178299755-108f03c2-bde0-4980-913e-e2297d8dff1c.png)

**Comment by abdelhai** [comment]
use firefox or this https://docs.deta.sh/docs/cli/auth#deta-access-tokens
please use github discussions not the issues tab

**Comment by killuazoldyckreal** [comment]
it's very simple, brave generally don't allow scripts from the websites. So you have to manually turn it on for that website like this:
 Click on the brave icon on the search bar=>Advanced Controls=>Block Scripts=>Turn Off
![image](https://user-images.githubusercontent.com/58257876/180602656-92a755a1-c3f0-4572-a69f-2fe69fc950fb.png)

**Reply by sudo-subg** [reply]
> Click on the brave icon on the search bar=>Advanced Controls=>Block Scripts=>Turn Off

Instead `allow all trackers and ads` fixed this issue for me.
